packages/components/pie-notification/src/index.ts

```diff
@@ -1,24 +1,193 @@
-import { LitElement, html, unsafeCSS } from 'lit';
-
-import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
+import {
+    LitElement,
+    unsafeCSS,
+    TemplateResult,
+    nothing,
+} from 'lit';
+import { StaticValue, html, unsafeStatic } from 'lit/static-html.js';
```

Comment:
Am I right in thinking that `StaticValue` can be imported as a type rather than a value? This should make it clearer that it can be omitted from the transpiled JS.
```suggestion
import { type StaticValue, html, unsafeStatic } from 'lit/static-html.js';
```